apps/platform/src/middleware.ts

```diff
@@ -0,0 +1,16 @@
+import { NextRequest, NextResponse } from 'next/server';
+
+export function middleware(request: NextRequest) {
```

Comment:
Good, though I think that URLs look better with language as part of the url, eg.

`muqa.org/en/projects` instead of `muqa.org/projects?lang=en`.

But we can leave it for now.

Reply:
we can use it like that, but than we have to add top parent directory [locale], something like that. We can also use subdomains or something else, but I ad like this only because that @tomislavmamic can change lang for presentation option for now.

it don't need to be final option